test/controllers/fhir/r4/api_controller_test.rb

```diff
@@ -950,9 +950,10 @@ def setup_patients
     assert_equal patient.user_defined_id_statelocal, json_response['identifier'].find { |i| i['system'].include? 'state-local-id' }['value']
   end
 
-  test 'should create History items when updating patient' do
+  test 'should create "Monitoring Change" and "Record Edit" History items when updating patient' do
```

Comment:
What about adding a negative test - for one of the changes we wouldn't want to see a history for?

Reply:
What's an example of a change we wouldn't want to see a history for?

Reply:
Well, taking a look a bit above in this PR's diff, `info_updates` is downselected using `!PatientHelper.monitoring_fields.include?(attr)`. I'd imagine any of of those would be an obvious choice.

Reply:
Ah I misunderstood what you meant. History items are still created for those things that are filtered out, just with a different history type. I'll adjust the testing to account for this.

Reply:
Refactored here https://github.com/SaraAlert/SaraAlert/pull/704/commits/a57d30b1f6b12ba30e763152825adff242fda3a1